flamby/datasets/fed_camelyon16/dataset_creation_scripts/tiling_slides.py

```diff
@@ -287,8 +359,15 @@ def main(batch_size, num_workers_torch, tile_from_scratch, remove_big_tiff, outp
     parser.add_argument(
         "--remove-big-tiff",
         action="store_true",
-        help="Whether or not to remove the original slides images that take \
-            up to 800G, after computing the features using them.",
+        help=(
+            "Whether or not to remove the original slides images that take "
+            "up to 800G, after computing the features using them."
+        ),
+    )
+    parser.add_argument(
+        "--use-phikon",
+        action="store_true",
+        help="Whether to use the phikon feature extractor",
```

Comment:
```suggestion
        help="Whether to use the much more performant phikon feature extractor trained with self-supervised learning on histology datasets from https://www.medrxiv.org/content/10.1101/2023.07.21.23292757v2 instead of imagenet-trained resnet",
```

Reply:
Make sure that the line isn't too long